src/app/components/shopping-cart/cart/cart.component.html

```diff
@@ -0,0 +1,9 @@
+<ul class="list-group">
+    <li class="list-group-item" *ngFor="let i of [1,2,3,4,5]">
```

Comment:
dynamic, not hard coded

Reply:
I have worked on this and will commit the code shortly.